wire-golden-files/src/main/kotlin/squareup/wire/mutable/MutablePayload.kt

```diff
@@ -93,7 +93,7 @@ public class MutablePayload(
 
   override fun toString(): String {
     val result = mutableListOf<String>()
-    if (preamble != null) result += """preamble=${sanitize(preamble!!)}"""
```

Comment:
If `preamble` is not null when would `preamble ?: ""` make sense?

Reply:
I think doing `?.let` would be best, but there's that compiler bug with them...

Reply:
Was the thought that it's confusing to read ? 

I considered one other thing which is to make `sanitize(...)` accept `String?` but I would have to rename the list variant to `sanitizeList` to disambiguate. Thought this was better than that.